static/bidder-info/adxcg.yaml

```diff
@@ -0,0 +1,13 @@
+maintainer:
+  email: "info@adxcg.com"
+capabilities:
+  app:
+    mediaTypes:
+      - banner
+      - video
+      - native
+  site:
+    mediaTypes:
+      - banner
+      - video
+      - native
```

Comment:
Does this bidder have a vendor id from the [IAB GDPR Vendor List](https://vendor-list.consensu.org/v2/vendor-list.json)?

Reply:
no